package.json

```diff
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
-    "cypress": "^12.16.0",
+    "cypress": "^13.2.0",
```

Comment:
@susannahmarcus , you might want to enable videos again. In this version they are turned off by default. Only screenshots are recorded.

```video: true```

Reply:
never mind, I see it.